README.md

```diff
@@ -27,6 +27,7 @@ Buildkite emoji will be shown on both light or dark backgrounds, and at a small
 
 Emoji | Aliases
 ----- | -------
+<img src="img-buildkite-64/github.png" width="20" height="20" alt="github"/> | `:clearbit:`
```

Comment:
I think `:clearbit:` is a copy-pasta typo.
Maybe best to put `:github:` next to `:octocat:` in the list — there doesn't seem to be any other sort criteria like alphabetical etc.

Reply:
Yeah, thanks - fixed the typo.

According to the readme, the sort order is newest at top. I'm tempted to leave GitHub at the top as, in the absence of any kind of changelog, it's an easy way to signify at a glance that we've recently updated it. We could pull octocat up to the top too though, given it's been recently updated?